### PR TITLE
Feat/pgraph testing

### PR DESCRIPTION
**Pull request by karthiksedoc**
* Fixed formatting issue of pgraph/pgraph_test.go
* Formatted commit message as guideline

### CODE REVIEW

File: pgraph/pgraph_test.go [hunk 1]
```diff
@@ -934,3 +934,11 @@ func TestSetValue(t *testing.T) {
 		t.Errorf("expecting value of %s at %s position, got %v", value, key, v)
 	}
 }
+
+func TestGetName(t *testing.T) {
+	G, _ := NewGraph("g9")
+	graph := G.GetName()
+	if graph == nil {
+		t.Errorf("Empty graph returned, expecting graph: %v", graph)
```
Comment (purpleidea) [inline pgraph/pgraph_test.go]: errors start with lower case messages

File: pgraph/pgraph_test.go [hunk 2]
```diff
@@ -934,3 +934,11 @@ func TestSetValue(t *testing.T) {
 		t.Errorf("expecting value of %s at %s position, got %v", value, key, v)
 	}
 }
+
+func TestGetName(t *testing.T) {
+	G, _ := NewGraph("g9")
+	graph := G.GetName()
+	if graph == nil {
```
Comment (purpleidea) [inline pgraph/pgraph_test.go]: inline this: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Comment (purpleidea) [inline pgraph/pgraph_test.go]: What type is `graph` here? =D

Comment (karthiksedoc) [reply pgraph/pgraph_test.go]: it's a short string, as per variable naming guideline change to s?

File: pgraph/pgraph_test.go (hunk 1 above)
Comment (karthiksedoc) [reply pgraph/pgraph_test.go]: thanks for info :) I'll correct this.

File: pgraph/pgraph_test.go (hunk 2 above)
Comment (karthiksedoc) [reply pgraph/pgraph_test.go]: this is sorted, thanks for pointing :)

**Pull request closed without merging**